README.md

```diff
@@ -138,7 +138,9 @@ if:
       - "config/.*"
 
   # "has_author_in" is satisfied if the user who opened the pull request is in
-  # the users list or belongs to any of the listed organizations or teams.
+  # the users list or belongs to any of the listed organizations or teams. The
```

Comment:
this could also be at a higher level, since it may apply in more cases. just adding it here because a user was asking